docs/typescript-plutus/api/lbf/build.nix

```diff
@@ -11,7 +11,7 @@ _:
     in
     {
       packages = {
-        inherit (tsFlake.packages) lbf-plutus-sample-project-typescript lbf-plutus-sample-project-typescript-tgz;
+        inherit lbf-plutus-sample-project-typescript;
```

Comment:
This is really breaking the mold, lbfPlutusTypescript is not supposed to return a flake like output, just a single derivation. Is there a reason why `tgz` is included, afact it can be constructed after?

I guess my question is: Why isn't lbfPlutusTypescript returning  only `lbf-plutus-sample-project-typescript` directly?

Reply:
> Is there a reason why tgz is included, afact it can be constructed after?

I suppose it can be. It's a bit painful to construct in the isolated build environment of nix but it can be done, and this is exactly the functionality that flake-lang.nix provides.

> Why isn't lbfPlutusTypescript returning only lbf-plutus-sample-project-typescript directly?

What do we mean by returning `lbf-plutus-sample-project-typescript` directly? Are we saying that we should only return the automatically generated files?

That won't work (unless the user manually adds a build step to compile the autogenerated lbf package) because the generated lbf package will be "invalid" in the sense that the automatically generated files contain no javascript files (it's a typescript project so it needs the build step to go from typescript to javascript) so all imports to the lbf package will fail.

Note that `npm` won't build the dependencies to depends on. As far as I can tell, `npm` / `node`  have no knowledge of any build step, and strictly speaking just look for javascript files when importing.

Reply:
For now, I'm going to merge this and get back to this later.